Vagrantfile

```diff
@@ -14,9 +14,9 @@ Vagrant.configure("2") do |config|
     test.vm.provider 'virtualbox' do |vb|
       # since opnsense expects nic1 (first) to be LAN, lets make it intnet
       # nic2 is wan, so we are doing nat ( routed to the host )
-      vb.customize ['modifyvm',:id, '--nic1', 'intnet', '--nic2', 'nat'] # swap the networks around
+      vb.customize ['modifyvm',:id, '--nic1', 'nat', '--nic2', 'nat'] # swap the networks around
```

Comment:
why are you chaning both not nat? LAN should be intnet since that is how we want to use a deployed gateway, don't we? We want the OPNsense instance to manage inbound nat entirely and not use host NAT to route around opnsense